tasks/browser-sync.js

```diff
@@ -5,9 +5,6 @@ const bs = require('browser-sync').create('SIM01'),
 
 module.exports = function() {
   if (config.server.url) {
-    bs.init({
-      proxy: config.server.url,
-      online: true,
```

Comment:
This line (10) has been removed, so I'd like to see it added to the example `Gulpfile.js`